app/src/main/java/io/apicurio/registry/storage/impl/sql/AbstractSqlRegistryStorage.java

```diff
@@ -1281,7 +1280,6 @@ public RuleType map(ResultSet rs) throws SQLException {
             throw new RegistryStorageException(e);
         }
         log.debug("Artifact rule row successfully inserted.");
```

Comment:
Probably not for this PR but I was wondering if it makes sense to return here something like the list of rows affected/created.

Reply:
What do you mean?  With this change there is now no return value.  But we'll get an exception thrown if there is an error.  If this method returned something like the # of rows, I think it would always return `1` or else throw an exception.  No?

Reply:
Sorry for the vague explanation. Added the comment in a bad place. In this case yes, it will always return `1` or throw an exception. I was thinking if it makes sense to add things like that as a confirmation, not only here but in other places as well. As I said, not for this PR but just something I was thinking about if it makes sense or if it will add some value or not.